tests/shell/shell_recorder/shell_recorder.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+@INCLUDE_COMMON@
```

Comment:
Iirc @KurtMi wanted the shell recorder to be directly executable. I am okay with `@INCLUDE_COMMON@` here, it makes sharing config snippets between other shell scripts and shell recorder easier.